backend/src/main/java/hu/flowacademy/vajdasagbrand/service/KeycloakClientService.java

```diff
@@ -51,10 +76,20 @@ public AccessTokenResponse login(String email, String password) {
                 .getAccessToken();
     }
 
+    public boolean enableUser(String username) {
+        UsersResource resource = keycloak.realm(keycloakPropertiesHolder.getKeycloakBackendClientRealm()).users();
+        List<UserRepresentation> listOfUsers = resource.search(username);
```

Comment:
Használd itt is a `stream().findFist()`-öt és a mapperbe rakd az update hívást.

Reply:
kijavítva